test/format/config.c

```diff
@@ -135,14 +137,20 @@ config_setup(void)
 			continue;
 
 		/*
-		 * Boolean flags are 0 or 1, but only set N in 100 where the
-		 * variable's min value is N.  Set the flag if we rolled >=
-		 * the min, 0 otherwise.
+		 * Boolean flags are 0 or 1, where the variables "min" value
+		 * is the percent chance the flag is "on" (so on if we rolled
+		 * <= N, otherwise off).
```

Comment:
"variable's."
At first it took a while to parse this. It needs consistent use of double-quotes around all the strings. Then it will read easier.

Reply:
Pushed in 632b009.